pkg/opni/commands/import_progress.go

```diff
@@ -126,7 +125,7 @@ func (model ProgressModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		}
 
 		model.percent = getProgressAsPercent(msg.Progress)
-		model.lastRead = msg.Progress.LastReadTimestamp
+		model.lastRead = msg.GetProgress().GetLastReadTimestamp()
```

Comment:
if `msg.GetProgress()` returns nil this'll panic. Typically this won't be an issue but see https://github.com/rancher/opni/blob/main/plugins/metrics/pkg/agent/runner.go#L399 and https://github.com/rancher/opni/blob/main/plugins/metrics/pkg/agent/runner.go#L404